corehq/apps/app_manager/dbaccessors.py

```diff
@@ -174,6 +174,17 @@ def get_brief_apps_in_domain(domain, include_remote=True):
     return apps
 
 
+def get_brief_app(domain, app_id):
+    from .models import Application
+    from corehq.apps.app_manager.util import get_correct_app_class
+    result = Application.get_db().view(
+        'app_manager/applications_brief',
+        key=[domain, app_id],
+    ).one(except_all=True)
+    doc = result['value']
+    return get_correct_app_class(doc).wrap(doc)
```

Comment:
this won't be an issue with the current code since the brief doc doesn't contain any of the fields referenced in the lazy migrations. It could be an issue in future if new lazy migration are added. 

Note that we already do this in the other brief doc views elsewhere in this file.